flow/project.py

```diff
@@ -1682,6 +1893,97 @@ def _generate_submit_script(self, _id, operations, template, show_template_help,
             self._show_template_help_and_exit(template_environment, context)
         return template.render(** context)
 
+    def submit_group(self, group, _id=None, env=None, parallel=False, flags=None,
```

Comment:
When we initially discussed this feature, I was under the impression that submitting a group should internally be not much different from submitting a normal operation, it just "bundles" them essentially. What is the justification of creating a completely separate top-level function for this?